src/main/java/com/feedzai/commons/sql/abstraction/engine/AbstractDatabaseEngine.java

```diff
@@ -882,6 +884,21 @@ public final synchronized Long persist(final String name,
         }
     }
 
+    /**
+     * Helper method that retrieves the {@link MappedEntity} from PDB cache.
+     *
+     * @param name The name of the {@link MappedEntity} to retrive.
+     * @return The {@link MappedEntity} from PDB cache.
+     * @throws DatabaseEngineException If the {@link MappedEntity} is not present in the cache.
+     */
+    protected MappedEntity getMappedEntityFromCache(final String name) throws DatabaseEngineException {
```

Comment:
Helper method that is used by all Database engines so they retrieve the Entity from PDB cache.